packages/account/src/Sections/Security/TwoFactorAuthentication/__tests__/two-factor-authentication-article.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import TwoFactorAuthenticationArticle from '../two-factor-authentication-article';
+
+describe('<TwoFactorAuthenticationArticle />', () => {
+    it('should render TwoFactorAuthenticationArticle component', () => {
+        render(<TwoFactorAuthenticationArticle />);
+
+        const two_factor_authentication_article = screen.getByText('Two-factor authentication (2FA)');
+        expect(two_factor_authentication_article).toBeInTheDocument();
+    });
+
+    it('should render article description properly', () => {
+        render(<TwoFactorAuthenticationArticle />);
+
+        const description = screen.getByText(
+            'Protect your account with 2FA. Each time you log in to your account, you will need to enter your password and an authentication code generated by a 2FA app on your smartphone.'
```

Comment:
lets use regex for this text

Reply:
@amina-deriv regex can be weak, right?
`220% Leverage` passes
- the test `/20% Leverage/`, 
- the test `/220% leverage/i`
going forward, do we really need to use regex in test files?